packages/wrangler/src/index.ts

```diff
@@ -608,7 +608,7 @@ export function createCLIParser(argv: string[]) {
 	// pipelines
 	wrangler.command(
 		"pipelines",
-		`🚰 Manage Worker Pipelines\n`,
+		`🚰 Manage Worker Pipelines ${chalk.hex(betaCmdColor)("[open beta]")}`,
```

Comment:
Still need the `\n` for (hacky) 'grouping' in --help output

Reply:
I think the output is fine. Should it look different to this?

```
wrangler pipelines

🚰 Manage Worker Pipelines [open beta]

COMMANDS
  wrangler pipelines create <pipeline>  Create a new pipeline
  wrangler pipelines list               List current pipelines
  wrangler pipelines show <pipeline>    Show a pipeline configuration
  wrangler pipelines update <pipeline>  Update a pipeline
  wrangler pipelines delete <pipeline>  Delete a pipeline

GLOBAL FLAGS
  -j, --experimental-json-config  Experimental: support wrangler.json  [boolean]
  -c, --config                    Path to .toml configuration file  [string]
  -e, --env                       Environment to use for operations and .env files  [string]
  -h, --help                      Show help  [boolean]
  -v, --version                   Show version number  [boolean]
```

Reply:
Not the subhelp (`wrangler pipelines`) but the top-level help (`wrangler --help`)